src/network/network_admin.cpp

```diff
@@ -172,7 +172,7 @@ NetworkRecvStatus ServerNetworkAdminSocketHandler::SendWelcome()
 	Packet *p = new Packet(ADMIN_PACKET_SERVER_WELCOME);
 
 	p->Send_string(_settings_client.network.server_name);
-	p->Send_string(_openttd_revision);
+	p->Send_string(_openttd_revision); // Not GetNetworRevisionString() since there is no field length limitation here
```

Comment:
GetNetworRevisionString
But, shouldn't it use it anyway, just for consistency if nothing else?

Reply:
Since this is the admin port and technically a separate protocol, my thought was no.